lib/src/base-styles/_nx-page.scss

```diff
@@ -23,25 +25,41 @@
     white-space: nowrap;
   }
 
-  .nx-btn-bar {
+  .nx-btn-bar, .nx-dropdown {
     grid-area: actions;
-    margin: 0 0 0 $nx-spacing-l;
+    margin: 0 0 0 $nx-spacing-xs;
   }
 }
 
-.nx-page-title__tags {
-  @include container-horizontal;
-
-  grid-area: tags;
-  margin-left: $nx-spacing-md;
+.nx-page-title__headings {
+  display: contents;
+}
 
-  .nx-policy-violation-indicator {
-    margin: 0;
-  }
+.nx-page-title__sub-title {
+  @include nx-truncate-ellipsis;
+  border-left: $nx-border-default;
+  margin: 0;
+  padding-left: $nx-spacing-md;
+  min-width: 150px;
 }
 
 .nx-page-title__description {
   @include container-vertical;
   grid-area: description;
-  margin: $nx-spacing-xs 0 0 0;
+  margin-top: $nx-spacing-xs;
+
+  & + .nx-page-title__tags {
+    margin-top: $nx-spacing-md;
+  }
+}
+
+.nx-page-title__tags {
+  @include container-horizontal;
+
+  grid-area: tags;
+  margin-top: $nx-spacing-xs;
+
+  .nx-policy-violation-indicator, .nx-tag {
```

Comment:
I worry about how specific this is.  We might have other things in the future that could fit here

Reply:
> I worry about how specific this is. We might have other things in the future that could fit here

Adjusted here: https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/8e28369200a8ae39884be11e3644cc402453e8c0